libsolidity/interface/Natspec.cpp

```diff
@@ -100,15 +100,52 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 		{
 			Json::Value method(devDocumentation(fun->annotation().docTags));
 			if (!method.empty())
+			{
 				// add the function, only if we have any documentation to add
+				Json::Value ret(Json::objectValue);
+
+				auto returnParams = fun->returnParameters();
+				auto returnDoc = fun->annotation().docTags.equal_range("return");
+
+				if (!returnParams.empty())
+				{
+					unsigned int n = 0;
+					for (auto i = returnDoc.first; i != returnDoc.second; i++)
+					{
+						string paramName = returnParams.at(n)->name();
+						string content = i->second.content;
+
+						if (paramName.empty())
+						{
+							paramName = "_" + std::to_string(n+1);
```

Comment:
Rethinking this again, I'd tend to start with `_0` as the first placeholder. That would probably be more inline with what people are expecting about array indices. I think you can just use `to_string(i)` then.